src/ui/connectionmanager.cc

```diff
@@ -184,12 +184,14 @@ void ConnectionManager::startLocalServer() {
   if (!check_file.exists()) {
     python_interpreter_executable = QString("py.exe");
   }
-#else
+#elif defined(Q_OS_MAC)
   QString python_interpreter_executable(qApp->applicationDirPath() + "/../Resources/veles-server/venv/bin/python3");
   QFileInfo check_file(python_interpreter_executable);
   if (!check_file.exists()) {
     python_interpreter_executable = QString("python3");
   }
+#else
+  #error OS not supported
```

Comment:
why? without this change it is chance that it will still work on other OS because of fallback to "python3"